### PR TITLE
Make Nutanix e2e optional

### PR DESCRIPTION
**Pull request by embik**
**What does this PR do / Why do we need it**:

Nutanix setup is still not available, so this makes Nutanix e2e tests optional for the time being.

**Does this PR close any issues?**:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

Signed-off-by: Marvin Beckers <marvin@kubermatic.com>

### CODE REVIEW

File: .prow/provider-nutanix.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@
 presubmits:
   - name: pre-kubermatic-e2e-nutanix-ubuntu-1.24
     decorate: true
+    optional: true
     run_if_changed: "(addons/csi/nutanix|pkg/provider/cloud/nutanix)"
```
Comment (xmudrii) [inline .prow/provider-nutanix.yaml]: If there's no Nutanix setup, we probably shouldn't run those tests at all, so I'd drop this `run_if_changed`.

Comment (embik) [reply .prow/provider-nutanix.yaml]: Good point. Done.

**Review by xmudrii — APPROVED** [review]

**Pull request merged**